src/harrastuspassi/harrastuspassi/api.py

```diff
@@ -308,9 +320,24 @@ def get_serializer_class(self):
         return self.serializer_class
 
 
+class PromotionFilter(filters.FilterSet):
+    exclude_past_events = filters.BooleanFilter(method='filter_past_events', label=_('Show upcoming only'))
```

Comment:
Is this named correctly?

Reply:
Well, seeing it's used elsewhere, I tried to give it a generic name, and I consider everything that has a beginning and an end an event.

Reply:
Alright, so event is a generic term and not referring to event instances. Got it.